lib/handlers/router-entities/route-caching/dynamo-route-caching-provider.ts

```diff
@@ -51,16 +59,33 @@ interface CachedRouteDbEntry {
   }
 }
 
+enum CachedRoutesSource {
+  CachedRoutes = 'CachedRoutes',
+  RoutesDb = 'RoutesDb',
+}
+
 const DEFAULT_TTL_MINUTES = 2
 export class DynamoRouteCachingProvider extends IRouteCachingProvider {
   private readonly ddbClient: DynamoDB.DocumentClient
   private readonly lambdaClient: Lambda
-  private readonly tableName: string
+  private readonly routesTableName: string
+  private readonly routesCachingRequestFlagTableName: string
+  private readonly cachingRoutesTableName: string
   private readonly cachingQuoteLambdaName: string
   private readonly cachingRequestFlagTableName: string
   private readonly ttlMinutes: number
 
+  private readonly ROUTES_DB_TTL = 24 * 60 * 60 // 24 hours
+  private readonly ROUTES_DB_FLAG_TTL = 2 * 60 // 2 minutes
+  private readonly DEFAULT_BLOCKS_TO_LIVE_ROUTES_DB = 2
+  private readonly DEFAULT_CACHEMODE_ROUTES_DB = CacheMode.Tapcompare
```

Comment:
It's because now when cachedRoutes is not there we will then use RoutesDB which is generalized, but I want to let the cache prime and try to measure the quote impact before going to Livemode. Eventually this will be Livemode.